allennlp/modules/seq2seq_encoders/__init__.py

```diff
@@ -6,17 +6,17 @@
 
 The available Seq2Seq encoders are
 
-- `"gru"` : https://pytorch.org/docs/master/nn.html#torch.nn.GRU
-- `"lstm"` : https://pytorch.org/docs/master/nn.html#torch.nn.LSTM
-- `"rnn"` : https://pytorch.org/docs/master/nn.html#torch.nn.RNN
+- `"gru"` : https://pytorch.org/docs/master/generated/torch.nn.GRU.html#torch.nn.GRU
+- `"lstm"` : https://pytorch.org/docs/master/generated/torch.nn.LSTM.html#torch.nn.LSTM
+- `"rnn"` : https://pytorch.org/docs/master/generated/torch.nn.RNN.html#torch.nn.RNN
 - `"augmented_lstm"` : allennlp.modules.augmented_lstm.AugmentedLstm
 - `"alternating_lstm"` : allennlp.modules.stacked_alternating_lstm.StackedAlternatingLstm
```

Comment:
I think this should be:

```suggestion
- `"alternating_lstm"` : allennlp.modules.seq2seq_encoders.StackedAlternatingLstmSeq2SeqEncoder
```